app/scripts/modules/core/src/presentation/icons/vectors/accordionExpand.svg

```diff
@@ -1,3 +1,7 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-  <path d="M15.086 8.258l-3.828 3.828 3.828 3.828-1.172 1.172-5-5 5-5z"/>
```

Comment:
@erikmunson @vigneshm I enbiggened the expand/collapse arrows, however they are not the full size of the borders because I opted for consistent sizing with `accordionExpandAll.svg` which does fill the bounding box.

Reply:
👍 seems reasonable, it's always going to be a tradeoff on related families of icons like this. Let's see how it looks and we'll adjust if needed